cli/cmd/config.go

```diff
@@ -29,6 +29,5 @@ var configCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(configCmd)
-
-	configCmd.Flags().StringVarP(&outputFileName, "file", "f", "", "Save content to local file")
+	configCmd.Flags().BoolVarP(&regenerateFile, "file", "f", false, "Save content to local file")
```

Comment:
I thought it will be nice to have -f to save to file, but wit can be achieved using " > filepath.yaml", so the only change that can be is to rename the flag because it will be better to call it regenerae IMO, but I will accept this as well